src/dialog-user/components/dialog-user/dialogField.html

```diff
@@ -125,6 +129,7 @@
               ng-disabled="vm.dialogField.read_only || vm.inputDisabled"
               ng-change="vm.changesHappened()"
               class="form-control"
+              data-container="body"
```

Comment:
Are we missing a pf-select here? Otherwise, data-container has no effect.

(same on line 145)